trecs/models/content.py

```diff
@@ -162,8 +162,6 @@ def __init__(  # pylint: disable-all
         if actual_item_representation is None:
             actual_item_representation = np.copy(item_representation)
 
-        measurements = [MSEMeasurement()]
```

Comment:
What will be returned if the user does not provide a measurement? An empty dataframe? If this is the case, would it be worth providing a warning here in the event that the user does not specify any measurement?

Reply:
Actually I found out. The `complete-guide.ipynb` now errors out when you run the method `get_measurements()` **after** running the simulation, which does not seem ideal to me. Seems like we should either add an assert during run to make sure there is some measurement added before running. Alternatively, we could issue a warning and return a dataframe full of `None` values for each timestep if no measurement is provided or something.

Reply:
I agree that something should be done since that is not an expected behavior. A warning when running the simulation would probably be the best option so that users can stop the simulation and fix the issue without wasting too long. Also returning `None` would sound good to me.